prow/cmd/checkconfig/main.go

```diff
@@ -448,6 +449,17 @@ func validate(o options) error {
 		}
 	}
 
+	// validate rerun commands match presubmit job triggering regex
+	for _, presubmits := range cfg.JobConfig.PresubmitsStatic {
+		for _, p := range presubmits {
+			if !cfg.Gerrit.IsAllowedPresubmitTrigger(p.RerunCommand) {
+				errs = append(errs, fmt.Errorf("rerun command %s in job %s does not conform to test command requirements,"+
+					"please make sure the trigger regex is a subset of %s and the rerun command matches the trigger regex",
+					p.RerunCommand, p.Name, presubmitTestRegex))
```

Comment:
We need to use the value from config here and below, but the message wording looks great though.